entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/mapper/RpcModeAnnotationMapper.kt

```diff
@@ -0,0 +1,22 @@
+package godot.entrygenerator.mapper
+
+import com.squareup.kotlinpoet.ClassName
+
+object RpcModeAnnotationMapper {
+    /**
+     * maps RpcModeAnnotations and RPCMode registration enums to ClassNames to provide import informations for kotlinPoet
+     */
+    fun mapRpcModeAnnotationToClassName(rpcModeAnnotationAsString: String): ClassName {
+        return when (rpcModeAnnotationAsString) {
+            "godot.registration.RPCMode.DISABLED" -> ClassName("godot.registration.RPCMode", "DISABLED")
+            "godot.registration.RPCMode.REMOTE" -> ClassName("godot.registration.RPCMode", "REMOTE")
+            "godot.registration.RPCMode.SYNC" -> ClassName("godot.registration.RPCMode", "SYNC")
```

Comment:
SYNC is deprecated.
I think deprecated should stay as they are not removed yet. But in this case, we miss SLAVE.
https://docs.godotengine.org/en/3.1/classes/class_multiplayerapi.html#enumerations

Reply:
SLAVE was renamed to PUPPET for godot 3.1 i think

Reply:
and i think we can remove the deprecated ones

Reply:
i removed the deprecated one if it's ok for you